MAIN/explainer.py

```diff
@@ -0,0 +1,62 @@
+import os
+import asyncio
+from PresentationParser import PresentationParser
+from GPTCommunicator import GPTCommunicator
+from SlideExplanation import SlideExplanation
+from PresentationExporter import PresentationExporter
+
+from dotenv import load_dotenv
+
+# Load environment variables from .env file
+load_dotenv()
+UPLOADS_FOLDER = os.getenv('UPLOADS_FOLDER_PATH')
+
+OUTPUTS_FOLDER = os.getenv('OUTPUTS_FOLDER_PATH')
+
+
+
+async def process_presentation(presentation_path, name_file):
```

Comment:
Use type hints and return types -
https://www.geeksforgeeks.org/duck-typing-in-python/